src/java/us/kbase/templates/module_dockerfile.vm.properties

```diff
@@ -1,4 +1,8 @@
+#if( $language == "python")
+FROM jjeffryes/kbasepythonsdk:latest
```

Comment:
We don't want to use a base image coming out of James' personal docker area. I have an image used for Python core services at kbase/kb_python:latest, maybe we should merge James' python3 SDK stuff into that image and just that one. Or else just put the python3 support into sdkbase2?

Reply:
I think we want to put it just into kbase/kb_python:latest because we want the default python interpreter to be Py3 in this image but we don't want that in sdkbase2. Do you have a branch I can submit a PR against for the kbase/kb_python dockerfile?

Reply:
This is my branch of sdkbase2 refactored for conda3: https://github.com/JamesJeffryes/sdkbase/tree/python

It looks like your kb_python branch does not have all the support libraries

Reply:
Sounds like it might be worth keeping a separate image/repo for kbasepythonsdk. Also was wondering if we really need a full clone of all those repos for SDK apps to run (I mean [these lines](https://github.com/JamesJeffryes/sdkbase/blob/python/Dockerfile#L107?))?

Reply:
I don't think the whole repo is being kept around - the repos are cloned into /tmp and then the necessary files copied out and then the repos are deleted. I would prefer that we packaged our kbase python modules as pip-installable modules, but we don't have that, so we have these hairballs.